hedyorg/hedy
# 📜 Use `{play}` template instead of literal `play` in music adventure

Currently when opening the music adventure level 1 in Nederlands you'll get the following instruction:

> In dit level leer je het `speel` commando te gebruiken om een muzieknoot te spelen!

Though in the example right next to it, it says:

```
play C4
...
```

This example is English and not Nederlands. It should say:

```
speel C4
...
```

Other levels of the music adventure _do_ correctly use the `speel` command. There `{play}` template is indeed used. I applied the same template to level 1.

Let me know if I should also create an issue for this PR.

**How to test**

Follow these steps to verify this PR works as intended:

* Go to http://localhost:5000/hedy/1#music
* Change the language to "Nederlands"
* Verify that the example uses `speel` instead of `play` commands.

**Checklist**
